### ISSUE TITLE
Unexpected debug logs in 1.2.0

### ISSUE THREAD

**Issue by PhilipDukhov** [saveourtool/diktat#1413]
## Describe the bug

I'm using gradle plugin with the following setup:
```
plugins {
    id("org.cqfn.diktat.diktat-gradle-plugin").version("1.2.0")
}

diktat {
    inputs {
        it.include("src/**/*.kt")
    }
}
```

Note that downgrading to 1.1.0 solves the issue

## Expected behavior

I expect to see only issues related strings in the output

## Observed behavior

I also see dozens of lines like this ones - looks like the count depends on processing files count:
```
09:32:05.102 [main] DEBUG org.cqfn.diktat.ruleset.rules.DiktatRuleSetProvider - Will run diktat-ruleset with diktat-analysis.yml (it can be placed to the run directory or the default file from resources will be used)
09:32:05.102 [main] DEBUG org.cqfn.diktat.common.config.rules.RulesConfigReader - Using diktat-analysis.yml file from the following path: .../diktat-analysis.yml
09:32:05.384 [Thread-0] DEBUG org.cqfn.diktat.ruleset.rules.DiktatRuleSetProvider - Will run diktat-ruleset with diktat-analysis.yml (it can be placed to the run directory or the default file from resources will be used)
09:32:05.385 [Thread-0] DEBUG org.cqfn.diktat.common.config.rules.RulesConfigReader - Using diktat-analysis.yml file from the following path: .../diktat-analysis.yml
```

## Steps to Reproduce

1. Open [attached sample project](https://github.com/saveourtool/diktat/files/9006798/MyApplication.zip)
2. run `./gradlew diktatCheck`

## Environment information
* diktat version: 1.2.0
* build tool (maven/gradle): gradle
* how is diktat run (CLI, plugin, etc.): plugin
* kotlin version: 1.7.0
* operating system: MacOS

**Comment by akuleshov7** [comment]
@PhilipDukhov thank you for this report! Looks like we made 1.2.0 release very dirty. Today or tomorrow we will releases 1.2.1 with several fixes, including this one

**Comment by petertrr** [comment]
Yes, we'll fix it in 1.2.1 by adding configuration for logback, and we have #1394 for more thorough investigation of logging, the problem is described in that issue
